cmd/monaco/deploy/deploy_test.go

```diff
@@ -191,6 +191,6 @@ func TestDeploy_ReportsErrorForBrokenV2Config(t *testing.T) {
 	manifestPath, _ := filepath.Abs("manifest.yaml")
 	_ = afero.WriteFile(testFs, manifestPath, []byte("manifestVersion: 1.0\nprojects:\n- name: project\nenvironmentGroups:\n- name: default\n  environments:\n  - name: environment1\n    url:\n      type: environment\n      value: ENV_URL\n    token:\n      name: ENV_TOKEN\n"), 0644)
 
-	err := Deploy(testFs, "manifest.yaml", []string{}, []string{}, true, false)
+	err := Deploy(testFs, "manifest.yaml", []string{}, "", []string{}, true, false)
 	assert.ErrorContains(t, err, "error while loading projects")
 }
```

Comment:
no new tests on this level?

Reply:
tbh I did not think about what, just wanted to trigger you to think if tests are needed here :wink: 
I didn't take a further look than the change set, and saw a few added/changed conditionals in deploy.go

Reply:
as discussed elsewhere, let's look into proper automated testing for environment deployments and overwrite on it's own in a followup